### PR TITLE
*: update version and fix compiling

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

**Comment by ngaut** [comment]
PTAL @BusyJay @siddontang

**Comment by zhangjinpeng1987** [comment]
LGTM

File: src/rocksdb_options.rs [hunk 1]
```diff
@@ -299,7 +299,7 @@ impl Options {
         }
     }
 
-    pub fn set_write_buffer_size(&mut self, size: size_t) {
+    pub fn set_write_buffer_size(&mut self, size: u64) {
```
Comment (siddontang) [inline src/rocksdb_options.rs]: should we use usize here ?

Comment (ngaut) [inline src/rocksdb_options.rs]: No, the compiler explicitly show the type.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**